src/presentation/test/integration/registration/name.test.ts

```diff
@@ -201,7 +201,7 @@ describe("Name Page", () => {
       const res = await request(app).get(NAME_URL);
 
       expect(res.status).toBe(200);
-      expect(res.text).toContain("What is the limited partnership name?");
+      expect(res.text).toContain(toEscapedHtml("What's the name of the partnership?"));
```

Comment:
should be referencing the key in the translation file instead of the value, namePage.whatIsName